pkg/utils/utils.go

```diff
@@ -450,3 +454,19 @@ func HasDefaultRoute(pciAddr string) (bool, error) {
 
 	return false, nil
 }
+
+// GetAuxNetDevicesFromPci returns list of auxiliary devices names for the specified PCI address
+func GetAuxNetDevicesFromPci(pciAddr string) ([]string, error) {
```

Comment:
why this is not moved to sriovnet?

Reply:
I can add it to sriovnet

Reply:
https://github.com/k8snetworkplumbingwg/sriovnet/pull/55